README.md

```diff
@@ -54,6 +54,11 @@ Community discussions happen primarily on the [dev mailing list][dev-list] or on
 
 Iceberg is built using Gradle 5.2.1.
 
+* To invoke a build and run tests: `./gradlew build`
+* To locally check sanity: `./gradlew check`
+* To skip tests: `./gradlew build -x test`
+* To skip tests and checkstyle (make sure you mean it): `./gradlew clean build -x test -x check`
```

Comment:
Let's remove this. I think it is clear how to skip steps from the `-x test` example. The note that checkstyle is run in CI and PRs will fail is needed if this is here, but seems needlessly long to me.

Reply:
Ok make sense